metrics/grafana/tidb.json

```diff
@@ -8013,7 +8013,7 @@
             "h": 7,
             "w": 8,
             "x": 16,
-            "y": 17
+            "y": 16
```

Comment:
Is it expected change?

Reply:
I think it doesn't matter.

Reply:
Why? It can change the position of the graph.